examples/publish_signals/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+
+	clarify "github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/query"
+	"github.com/clarify/clarify-go/views"
+)
+
+const (
+	// Annotation keys should be prefixed to avoid collision. We recommend:
+	// <company domain or GitHub org>/<software name>/<variable name>
+	keyExampleName          = "clarify/clarify-go/example/name"
+	keyTransform            = "clarify/clarify-go/example/transform"
+	keySignalAttributesHash = "clarify/clarify-go/example/source-signal/attributes-hash"
+	keySignalID             = "clarify/clarify-go/example/source-signal/id"
+
+	// The value to describe our transformVersion; incrementing or changing this
+	// number forces items to be republished.
+	transformVersion = "v1"
+)
+
+func main() {
+	creds, err := clarify.CredentialsFromFile("clarify-credentials.json")
+	if err != nil {
+		panic(err)
+	}
+
+	ctx := context.Background()
+	client := creds.Client(ctx)
+
+	// To select or publish signals, you must enable access to the "admin"
+	// namespace in clarify; this allows selecting and publishing signals from all integrations
```

Comment:
In my case it works only when both `admin` and `clarify` namespaces enabled. Is it intended by the docstring?

Reply:
It's not intended; might be a bug in the API. I will double check it.

Reply:
I found the bug in the API; let's close this issue from an SDK point of view; the fix needs to go into the API.